README.md

```diff
@@ -44,7 +44,7 @@ Documentation
     add(data)
 
 Adds *data* to the estimator where data is a string, buffer, or bytes
-type.
+type. Returns `True` if the approximated cardinality estimated by the HyperLogLog changed.
```

Comment:
It's not strictly True that the cardinality estimate changes when a register is updated. If the registers are sparsely populated and a linear count is used then updating a register doesn't necessarily change the cardinality estimate (e.g. if the register value goes from 5 to 10 the cardinality under a linear count is the same). It would be better to simply say:
```suggestion
type. Returns `True` if the registers were updated otherwise returns `False`.
```

Reply:
I had tried to use the same text that redis uses in its [documentation](https://redis.io/commands/pfadd) figuring if it was good enough for redis it could be good enough here. 
https://github.com/antirez/redis/blob/1d13ff0b397797b34973b4c33b0fd60e8ca4bccc/src/hyperloglog.c#L490-L492

But yes, agreed that your suggestion is absolutely correct.